spec/system/jobseekers_can_search_for_jobs_spec.rb

```diff
@@ -116,6 +116,19 @@
     end
   end
 
+  context "jobseekers can sort jobs by closing date" do
+    it "lists the most jobs with the earliest closing date first" do
```

Comment:
I think this is a typo:
```suggestion
    it "lists the jobs with the earliest closing date first" do
```